src/app/examples/divider-example/divider-example.component.html

```diff
@@ -0,0 +1,5 @@
+<kirby-card [hasPadding]="true" [themeColor]="themeColor">
+  <h1>Above header</h1>
```

Comment:
A bit confusing as we also have card headers.... 🤔 
Maybe:
```suggestion
  <h1>Above divider</h1>
```

Reply:
Typo... :)